includes/data/connection/class-downloadable-item-connection-resolver.php

```diff
@@ -186,4 +215,18 @@ public function get_items() {
 	public function is_valid_offset( $offset ) {
 		return 'string' === gettype( $offset );
 	}
+
+	/**
+	 * Validates Model.
+	 *
+	 * If model isn't a class with a `fields` member, this function with have be overridden in
+	 * the Connection class.
+	 *
+	 * @param array $model Downloadable item model.
+	 *
+	 * @return bool
+	 */
+	protected function is_valid_model( $model ) {
```

Comment:
```suggestion
	protected function is_valid_model( array $model ) {
```

Reply:
That's an overridden [function](https://github.com/wp-graphql/wp-graphql/blob/develop/src/Data/Connection/AbstractConnectionResolver.php#L557), adding typehinting actually breaks the class.